common/content/buffer.js

```diff
@@ -764,12 +764,6 @@ const Buffer = Module("buffer", {
             win.scrollBy(0, win.innerHeight / 2 * direction);
     },
 
-    _scrollByScrollSize: function _scrollByScrollSize(count, direction) {
-        if (count > 0)
-            options["scroll"] = count;
```

Comment:
Sorry, it seems I was a bit hasty with my commit. 

I interpreted (and use) `options["scroll"]` as constant, changing what is default or set by a rc file by doing `3<C-d>`, and then "breaking" the lines scrolled by the next `<C-d>` didn't make sense to me. Although now I understand this is intended behavior.

Wouldn't it still possibly be better if this acted as a contant option, not being changed by the count? This would better match how other scroll mappings work, using the count to update can be a bit confusing. 

I checked, and the only function using `scrollByScrollSize(direction, count)` was `_scrollByScrollSize(count, direction)`.